helpdesk.com.custom-domain.json

```diff
@@ -0,0 +1,39 @@
+{
+  "providerId": "helpdesk.com",
+  "providerName": "HelpDesk",
+  "serviceId": "custom-domain",
+  "serviceName": "HelpDesk custom domain setup",
+  "version": 1,
+  "logoUrl": "https://helpdesk.design/images/helpdesk/DIGITAL%20%28RGB%29/PNG/Logo_RGB_Green.png",
+  "description": "Verifies domain ownership and adds records to allow using the domain in HelpDesk",
+  "variableDescription": "helpdeskId: customer's ID in HelpDesk; dkimKeyName: DKIM key name; dkimKeyValue: DKIM key value",
+  "syncBlock": false,
+  "sharedProviderName": false,
+  "sharedServiceName": false,
+  "syncPubKeyDomain": "helpdesk.com",
+  "syncRedirectDomain": "app.helpdesk.com,app.labs.helpdesk.com",
+  "multiInstance": true,
```

Comment:
why `multiInstance=true`?
the CNAME on `"host": "pm-bounces"` would always render conflicts between instances, so effectively there will be only one (unless 2nd instance is only with groupId `ownershipVerification`)

Reply:
In theory a domain could be connected to two customer accounts each with separate domain key and verification record. The CNAME for pm-bounces could be common for the two installations. But if this poses a problem, I can change multiInstance to false.

Reply:
Changed.